src/main/webapp/availability.html

```diff
@@ -44,6 +44,78 @@ <h5 class="text-center lead">Update your availability to be an Interviewer for t
       <!--TODO: Include Timezone consideration; Check inputs for correctness: end is after start, at least an hour long;
          Add pagination; Figure out how to update the dates (dynamically); store and show inputted data; Allow for more than one
          time range per day (have a plus button)-->
+      <h5 class="font-weight-bold">Sunday 6/28</h5>
```

Comment:
These changes don't look related to using autovalue....

I don't want to keep pushing back on these "extra" changes (because I know it requires another whole feedback loop before this can get submitted), but why is it happening?

Reply:
l don't know, it came from commit 4c721bf, which were just tweaks. l'll try to be more careful and pinpoint why if it ever happens again.

Reply:
I think they're still here.

What's going on?